internal/services/appconfiguration/app_configuration_data_source.go

```diff
@@ -93,6 +94,29 @@ func dataSourceAppConfiguration() *pluginsdk.Resource {
 				Computed: true,
 			},
 
+			"replica": {
+				Type:     pluginsdk.TypeSet,
```

Comment:
We can't use sets in data sources because every item gets the same hashcode. This can just be a computed list and should be fine.

Reply:
fixed, thanks.